charts/tfy-llm-gateway-infra/templates/_helper.tpl

```diff
@@ -0,0 +1,47 @@
+{{/*
+Clickhouse labels
+*/}}
+{{- define "clickhouse.labels" -}}
+{{- if .Values.clickhouse.labels }}
+{{ toYaml .Values.clickhouse.labels }}
+{{- else if .Values.global.labels }}
+{{- toYaml .Values.global.labels }}
+{{- else }}
+{}
+{{- end }}
+{{- end }}
+
+{{/*
+Clickhouse annotations
+*/}}
+{{- define "clickhouse.annotations" -}}
+{{- if .Values.clickhouse.annotations }}
+{{- toYaml .Values.clickhouse.annotations }}
+{{- else if .Values.global.annotations }}
+{{- toYaml .Values.global.annotations }}
+{{- end }}
+{{- end }}
+
+{{/*
+Clickhouse virtualservice annotations
+*/}}
+{{- define "clickhouse.virtualservice.annotations" -}}
+{{- if .Values.clickhouse.annotations }}
+{{- toYaml .Values.clickhouse.annotations }}
+{{- else if .Values.global.annotations }}
+{{- toYaml .Values.global.annotations }}
+{{- else }}
+{}
+{{- end }}
+{{- end }}
+
```

Comment:
Where is "nats.labels" - is this chart even working ?

Reply:
There is a nats.labels in the nats charts dependency which was conflicting with the one in the helpers function. This would reference the existing one.